### PR TITLE
Transliteration

### PR DESCRIPTION
**Pull request by developer-aditya**
[RCF-264](https://mosip.atlassian.net/browse/RCF-264) - Transliteration for every textbox having multi language input added

[RCF-264]: https://mosip.atlassian.net/browse/RCF-264?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: android/app/src/main/java/io/mosip/registration_client/api_services/TransliterationApi.java
Comment (jainhitesh9998) [inline android/app/src/main/java/io/mosip/registration_client/api_services/TransliterationApi.java:1]: can we segregrate the transliteration logic 

have a similar interface and implementation separated and use pigeon api's just to call out the service
https://github.com/mosip/commons/blob/master/kernel/kernel-core/src/main/java/io/mosip/kernel/core/transliteration/spi/Transliteration.java

**Review by jainhitesh9998 — CHANGES_REQUESTED** [review]
can we segregrate the transliteration logic

File: android/app/src/main/java/io/mosip/registration_client/api_services/TransliterationApi.java
Comment (developer-aditya) [reply android/app/src/main/java/io/mosip/registration_client/api_services/TransliterationApi.java:1]: okay sure

File: android/transliterationmanager/src/test/java/io/mosip/registration/transliterationmanager/ExampleUnitTest.java
Comment (jainhitesh9998) [inline android/transliterationmanager/src/test/java/io/mosip/registration/transliterationmanager/ExampleUnitTest.java:1]: we should have relevent test case for the libraries, not example ones

File: android/transliterationmanager/src/test/java/io/mosip/registration/transliterationmanager/TransliterationServiceUnitTest.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import io.mosip.registration.transliterationmanager.service.TransliterationServiceImpl;
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+
+public class TransliterationServiceUnitTest {
+    TransliterationServiceImpl transliterationService;
```
Comment (jainhitesh9998) [inline android/transliterationmanager/src/test/java/io/mosip/registration/transliterationmanager/TransliterationServiceUnitTest.java:17]: we should initialize interface, not the impl here

**Review by jainhitesh9998 — CHANGES_REQUESTED** [review]

**Review by jainhitesh9998 — APPROVED** [review]

**Pull request merged**